templates/blog/post-detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <a href="{% url 'blog:home' %}" class="home-button">Home</a>
+  <div class="container">
+    <div class="card my-2">
+      <div class="card-body">
+        <h3><strong>{{ post.title }}</strong></h3>
+        <p class="text-muted small">{{ post.owner.first_name }} {{ post.owner.last_name }}<br>{{ post.created_time }}</p>
+        <p class="text-muted small"></p>
+        <div class="card my-2">
+          <div class="card-body">
+            <p class="post-content">{{ post.content }}</p>
+          </div>
+        </div>
+        <br>
+        <p><strong>{{ comments.count }} comment{{ comments.count|pluralize:",s" }}</strong></p>
+      {% for comment in comments.all %}

```

Comment:
The comments loop should use 'comments' instead of 'comments.all'. The 'comments' variable is already a queryset, so '.all' is unnecessary.